### PR TITLE
ops-zagg-client

### PR DESCRIPTION
**Pull request by mwoodson**

### CODE REVIEW

File: openshift_tools/monitoring/zagg_sender.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    from openshift_tools.monitoring.zagg_sender import ZaggSender, ZaggHeartbeat
+    HOSTNAME = 'use-tower1.ops.rhcloud.com'
+    METRICS = [
+        'kernel.all',
+        'swap.free',
+        'swap.length',
+        'swap.used',
+        ]
+
+    ZAGGCONN = ZaggConnection(host='172.17.0.151', user='admin', password='r3dh@t')
```
Comment (twiest) [inline openshift_tools/monitoring/zagg_sender.py]: can we just do 'pass' for the password?

So it'd be this:
```
ZAGGCONN = ZaggConnection(host='172.17.0.151', user='admin', password='pass')
```

File: openshift_tools/monitoring/zagg_sender.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    ZAGGHEARTBEAT = (templates=['template1', 'template2'], hostgroups=['hostgroup1', 'hostgroup2'])
+
+    zs = ZaggSender(host=HOSTNAME, zagg_connection=ZAGGCONN)
+    zs.add_pcp_metrics(METRICS)
+    zs.add_heartbeat(ZAGGHEARTBEAT)
+    zs.add_zabbix_keys({ 'test.key' : '1' })
+    zs.send_metrics_to_zagg()
```
Comment (twiest) [inline openshift_tools/monitoring/zagg_sender.py]: Since this is the zagg_sender, I'd prefer this to be:

```
zs.send_metrics()
```

**Comment by mwoodson** [comment]
ok to test

File: openshift_tools/monitoring/zagg_sender.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+        zabbix_metrics = []
+
+        for key, value in zabbix_keys.iteritems():
+            zabbix_metric = UniqueMetric(host, key, value)
+            zabbix_metrics.append(zabbix_metric)
+
+        self.unique_metrics += zabbix_metrics
+
+    def send_metrics_to_zagg(self):
```
Comment (twiest) [inline openshift_tools/monitoring/zagg_sender.py]: same as above

File: openshift_tools/monitoring/zagg_sender.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    zs.add_zabbix_keys({ 'test.key' : '1' })
+    zs.send_metrics()
+"""
+
+from openshift_tools.monitoring.zagg_client import ZaggClient
+from openshift_tools.monitoring import pminfo
+from openshift_tools.monitoring.metricmanager import UniqueMetric
+from collections import namedtuple
+
+ZaggHeartbeat = namedtuple("ZaggHeartbeat", ["templates", "hostgroups"])
```
Comment (twiest) [inline openshift_tools/monitoring/zagg_sender.py]: this should come from zagg_common

**Comment by twiest** [comment]
Looks great! :+1:

**Pull request merged**